lib/util/ui.js

```diff
@@ -1,59 +1,11 @@
+import coreUtils from '@opentripplanner/core-utils'
+
 import { MainPanelContent } from '../actions/ui'
-import { summarizeQuery } from './query'
 import { getActiveSearch } from './state'
 
 // Set default title to the original document title (on load) set in index.html
 const DEFAULT_TITLE = document.title
```

Comment:
As mentioned elsewhere, I think this stuff can be consolidated into `state`. I think it's a more appropriate location and simplifies things.